templates/registration/logg_out.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
```

Comment:
Typo Issue: The file name `logg_out.html` seems to have a typo. It should be `logged_out.html` to correctly reflect the action of logging out.